builtin/mainmenu/textures.lua

```diff
@@ -145,6 +145,17 @@ function mm_texture.set_game(identifier,gamedetails)
 	
 	local path = gamedetails.path .. DIR_DELIM .."menu" ..
 									 DIR_DELIM .. identifier .. ".png"
+	if core.setting_getbool("random_menu_images") == true then
+		local count = gamedetails.image_count
+		if tonumber(count) == nil then
```

Comment:
The C++ code uses a `std::string` to pass a number around, it would be easier to just pass the number itself.